docs/start/index.mdx

```diff
@@ -10,7 +10,8 @@ in_page_toc_max_heading_level: 2
 
 The quantum community has explored quantum computing on the cloud since 2016.  Now the community has a new challenge: *demonstrate useful quantum computation*. 
 
-Through IBM Quantum, you have access to performant and utility-scale (>100-qubit) quantum systems for your work, as well as scalable and flexible quantum software. Whether you try out a no-cost plan or run your programs through a pay-as-you-go or premium plan, you can start doing useful work with quantum computing now.
+Through IBM Quantum, you have access to performant and utility-scale (>100-qubit) quantum systems for your work, as well as scalable and flexible quantum software with the milestone [1.0 release](https://research.ibm.com/blog/qiskit-1-launch) of Qiskit, which has ushered in a new era of performance and stability for the Qiskit community. Whether you try out a no-cost plan or run your programs through a pay-as-you-go or premium plan, you can start doing useful work with quantum computing now.
```

Comment:
```suggestion
Through IBM Quantum, you have access to performant and utility-scale (>100-qubit) quantum systems for your work, as well as scalable and flexible quantum software. In particular, the milestone [1.0 release](https://research.ibm.com/blog/qiskit-1-launch) of Qiskit has ushered in a new era of performance and stability for the quantum software community. Whether you try out a no-cost plan or run your programs through a pay-as-you-go or premium plan, you can start doing useful work with quantum computing now.
```

I realised that we should probably decouple Qiskit from the "scalable and flexible quantum software" line, because there is more quantum software in docs than just qiskit and we don't want to leave those other things out. how about something like this instead?